await/src/main/java/com/adyen/checkout/await/internal/ui/model/AwaitOutputData.kt

```diff
@@ -5,10 +5,12 @@
  *
  * Created by caiof on 25/8/2020.
  */
-package com.adyen.checkout.await
+package com.adyen.checkout.await.internal.ui.model
 
+import androidx.annotation.RestrictTo
 import com.adyen.checkout.components.base.OutputData
 
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Can't this be internal?

Reply:
This is used inside `AwaitDelegate` and I need to make `AwaitDelegate` internal to make it work and I can't make the delegate internal.